tools/vfscount

```diff
@@ -16,47 +16,9 @@ from ctypes import c_ushort, c_int, c_ulonglong
 from time import sleep, strftime
 from sys import stderr
```

Comment:
With this cleanup, ctypes, sys, and strftime can be removed from the import list.